css/style.css

```diff
@@ -271,7 +273,11 @@ input[type="checkbox"] {
 }
 
 .vis-content {
-  position: relative;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
```

Comment:
Can you make this overflow: hidden;  also or solve why there are scroll bars perhaps?